### DISCUSSION TITLE
I am unable to properly apply for student developer pack.

### DISCUSSION THREAD

**Discussion by EncryptMan** [community/community#61064] [GitHub Education]
### Hi everyone,

In my student card date is written in back. I am unable to upload two pictures of my card from front and back. There is only option to upload one photo. I got rejected. Github want to see my name and date on my student card.

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
